build.gradle.kts

```diff
@@ -118,12 +130,9 @@ allprojects {
 
         repositories {
             maven {
-                name = "GitHubPackages"
-                setUrl("https://maven.pkg.github.com/Black-Kamelia/Sprinkler")
-                credentials {
-                    username = localProps["githubUsername"] as String? ?: "Unknown user"
-                    password = localProps["githubPassword"] as String? ?: "Unknown password"
-                }
+                name = "mavenCentral"
+                credentials(PasswordCredentials::class)
+                url = uri("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/")
```

Comment:
extract the url and get it to be conditionally that of the snapshot repository depending on the version of the module